GameDataParser/Parsers/EnchantLimitParser.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Xml;
+using GameDataParser.Files;
+using Maple2.File.IO.Crypto.Common;
+using Maple2Storage.Enums;
+using Maple2Storage.Types;
+using Maple2Storage.Types.Metadata;
+
+namespace GameDataParser.Parsers;
+
+public class EnchantLimitParser : Exporter<List<EnchantLimitMetadata>>
+{
+    public EnchantLimitParser(MetadataResources resources) : base(resources, MetadataName.EnchantLimit) { }
+
+    protected override List<EnchantLimitMetadata> Parse()
+    {
+        List<EnchantLimitMetadata> items = new();
+
+        foreach (PackFileEntry entry in Resources.XmlReader.Files)
+        {
+            if (!entry.Name.StartsWith("table/enchantlimittable"))
+            {
+                continue;
+            }
```

Comment:
If its only one file that needs parsing, can we use a FirstOrDefault instead of ForEach?